CRM/Case/Form/Activity/ChangeCaseStatus.php

```diff
@@ -105,10 +105,7 @@ public static function buildQuickForm(&$form) {
 
     foreach ($form->_defaultCaseStatus as $keydefault => $valdefault) {
       if (!array_key_exists($valdefault, $form->_caseStatus)) {
-        $form->_caseStatus[$valdefault] = CRM_Core_OptionGroup::getLabel('case_status',
-          $valdefault,
-          FALSE
-        );
+        $form->_caseStatus[$valdefault] = CRM_Core_PseudoConstant::getLabel('CRM_Case_BAO_Case', 'case_status', $valdefault);
```

Comment:
I believe this is incorrect and should be 'status_id' instead of 'case_status'

Reply:
Yes